### PR TITLE
Add Azure Stack HCI workbook

### PR DESCRIPTION
**Pull request by myunghojung**
## PR Checklist

* [x] Explain your changes, so people looking at the PR know *what* and *why*, the code changes are the *how*.

This change adds a template for Azure Stack HCI Insights. The dashboard shows performance, health and usage of registered Azure Stack HCI clusters that are connected to Azure and are enrolled in monitoring.

image
image
image
image
image

### If adding or updating templates:
* [x] post a screenshot of templates and/or gallery changes
* [x] ensure your template has a corresponding gallery entry in the gallery folder
* [x] If you are adding a new template, add your team and template/gallery file(s) to the CODEOWNERS file. CODEOWNERS entries should be teams, not individuals
* [x] ensure all steps have meaningful names
* [x] ensure all parameters and grid columns have display names set so they can be localized
* [x] ensure that parameters id values are unique __or they will fail PR validation__ (parameter ids are used for localization)
* [x] ensure that steps names are unique __or they will fail PR validation__ (step names are used for localization)
* [x] grep `/subscription/` and ensure that your parameters don't have any hardcoded resourceIds __or they will fail PR validation__
* [x] remove `fallbackResourceIds` and `fromTemplateId` fields from your template workbook __or they will fail PR validation__

### CODE REVIEW

File: Workbooks/Azure Stack HCI/AtScale/Insights.workbook [hunk 1]
```diff
@@ -0,0 +1,2056 @@
...
+                              "text": ""
+                            }
+                          ]
+                        }
+                      },
+                      "showBorder": false,
+                      "size": "auto"
+                    }
+                  },
+                  "name": "query - 1"
```
Comment (yves-chan) [inline Workbooks/Azure Stack HCI/AtScale/Insights.workbook:567]: a lot of these names have `query - 1`, `text - 1`, etc. You can consider giving these groups and steps meaningful names

File: Workbooks/Azure Stack HCI/AtScale/Insights.workbook [hunk 2]
```diff
@@ -0,0 +1,2056 @@
...
+        "parameters": [
+          {
+            "id": "2c3bd553-1307-4363-97c7-e2b1bc3d543e",
+            "version": "KqlParameterItem/1.0",
+            "name": "timeRange",
+            "label": "Time range",
+            "type": 4,
+            "isRequired": true,
+            "value": {
+              "durationMs": 604800000
```
[the remark this answers is not in the document]
Comment (myunghojung) [reply Workbooks/Azure Stack HCI/AtScale/Insights.workbook:20]: HCI log events frequency is 1 hour by default so we decided to set default time range to 7 days to show meaningful trends in line graphs. Also, only the first 10 clusters will be selected by default. Could it still cause performance issue? Then, I'll check with team and update.

File: Workbooks/Azure Stack HCI/AtScale/Insights.workbook (hunk 1 above)
Comment (myunghojung) [reply Workbooks/Azure Stack HCI/AtScale/Insights.workbook]: I renamed all of them.

File: Workbooks/Azure Stack HCI/AtScale/Insights.workbook (hunk 2 above)
[the remark this answers is not in the document]
Comment (yves-chan) [reply Workbooks/Azure Stack HCI/AtScale/Insights.workbook:20]: I see, that should be fine then

File: Workbooks/Azure Stack HCI/AtScale/Insights.workbook [hunk 3]
```diff
@@ -0,0 +1,2028 @@
...
+            "isRequired": true,
+            "multiSelect": true,
+            "quote": "'",
+            "delimiter": ",",
+            "query": "Resources\r\n| where type =~ 'microsoft.azurestackhci/clusters'\r\n| summarize by subscriptionId\r\n| union (Resources\r\n| where type =~ 'microsoft.azurestackhci/clusters'\r\n| summarize Count = count() by subscriptionId\r\n| top 1 by Count desc)\r\n| summarize Count = count() by subscriptionId\r\n| project value = subscriptionId, label = subscriptionId, selected = Count > 1",
+            "crossComponentResources": [
+              "value::selected"
+            ],
+            "value": [
+              "/subscriptions/680d0dad-59aa-4464-adf3-b34b2b427e8c"
```
Comment (yves-chan) [inline Workbooks/Azure Stack HCI/AtScale/Insights.workbook]: need to remove hardcoded subs

**Review by yves-chan — APPROVED** [review]

File: Workbooks/Azure Stack HCI/AtScale/Insights.workbook (hunk 3 above)
Comment (myunghojung) [reply Workbooks/Azure Stack HCI/AtScale/Insights.workbook]: I just removed it. Thanks!

**Review by muthus-msft — APPROVED** [review]

**Pull request merged**